opensrp-chw/src/kk/resources/child_hv_pmtct.properties

```diff
@@ -12,12 +12,12 @@ child_hv_pmtct.step1.hiv_status.label = Enter HIV status
 child_hv_pmtct.step1.hiv_status.options.chk_hiv_status_positive.text = HIV-positive
 child_hv_pmtct.step1.hiv_status.options.chk_hiv_status_negative.text = HIV-negative
 child_hv_family_member_involvement.step1.advice_mother_next_checkup.text = CHW Information: Advice the mother to go for the next checkup when the child is 18 months
-child_hv_family_member_involvement.step1.safe_sex_practices.text = CHW Information: Remind mother/caregiver of the importance of safe sex practices, and ensuring both she and her partner are tested and know their status so they can protect themselves and their unborn children.
+child_hv_family_member_involvement.step1.safe_sex_practices.text = CHW Information: Remind mother of the importance of safe sex practices, and ensuring both she and her partner are tested and know their status so they can protect themselves and their unborn children.
 child_hv_pmtct.step1.already_taking_art.label = Are you already taking AntiRetroviralTheraphy (ART)?
 child_hv_pmtct.step1.already_taking_art.options.taking_art_yes.text = Yes
 child_hv_pmtct.step1.already_taking_art.options.taking_art_no.text = No
 child_hv_family_member_involvement.step1.encourage_mother_continue_treatment.text = Encourage the mother to continue with her treatment.
-child_hv_family_member_involvement.step1.nearest_ctc.text = Encourage mother/caregiver to go to the nearest Care, Treatment, and Counselling (CTC) /PMCTC clinic.
+child_hv_family_member_involvement.step1.nearest_ctc.text = Encourage mother to go to the nearest Care, Treatment, and Counselling (CTC) /PMCTC clinic.
```

Comment:
@devcolonel Also remove the word "mlezi" for swahili translation

Reply:
Hi @maulidkategile, since mario is off do you ming picking this up so that it doest get stuck

Reply:
@issyzac Noted